### PR TITLE
Fix x64 build due to Android project mappings.

### PR DESCRIPTION
**Pull request by JunielKatarn**
Re-introduced this small configuration fix to unblock x64 builds.
(See 49939eae0bf6468626d64354ef7f1c512b0118b1).

### CODE REVIEW

File: Source/Android/Bridge/Bridge.vcxproj [hunk 1]
```diff
@@ -5,10 +5,18 @@
       <Configuration>Debug</Configuration>
       <Platform>Win32</Platform>
     </ProjectConfiguration>
+    <ProjectConfiguration Include="Debug|x64">
```
Comment (project64) [inline Source/Android/Bridge/Bridge.vcxproj]: this project does not need to be in 64bit

File: Source/Android/PluginInput/PluginInput.vcxproj [hunk 2]
```diff
@@ -5,10 +5,18 @@
       <Configuration>Debug</Configuration>
       <Platform>Win32</Platform>
     </ProjectConfiguration>
+    <ProjectConfiguration Include="Debug|x64">
+      <Configuration>Debug</Configuration>
```
Comment (project64) [inline Source/Android/PluginInput/PluginInput.vcxproj]: this project does not need to be in 64bit

File: Source/Android/Bridge/Bridge.vcxproj (hunk 1 above)
Comment (JunielKatarn) [inline Source/Android/Bridge/Bridge.vcxproj]: @project64 

I believe it doesn't hurt to have it (there could eventually be an x64 Android variant).
Plus, without this change, the overall build fails for that platform.

Another alternative would be to fully leave out Android projects on x64 builds.

Which do you think is best?
In any case, we need to submit a change or builds will keep failing.

**Comment by project64** [comment]
how is it failing, I do not get any errors when I compile it

**Comment by JunielKatarn** [comment]
I'm only getting the errors when running from the command line (using `MSBuild`).
It looks like Visual Studio resolves the configuration mappings differently, safely ignoring the Android projects in x64.

MSBuild is relevant for automated and unattended builds (meaning, not needing to open the IDE just to build).
Would it hurt to include the Android plugins in x64? If so, I'll keep researching how to have both IDE and command line builds working.

**Comment by project64** [comment]
what is the command line you using

**Comment by JunielKatarn** [comment]
> what is the command line you using

`MSBuild Project64.sln /p:Configuration=Release /p:Platform=x64`
(Note, this behavior exists at least in VS2012 and later).

**Comment by JunielKatarn** [comment]
@project64 
I have updated the CR. It now skips the plugins on x64, instead of explicitly building them.
Both IDE-based and command line-based builds succeed.

**Pull request merged**